ckine/model.py

```diff
@@ -264,3 +325,21 @@ def getRateVec(rxntfr):
         FullRateVec = np.array(list(ratesParamsDict.values()), dtype=np.float)
 
     return FullRateVec
+
+
+def mut_adjust(rxntfr, mutdict, mut_name):
+    """Adjust alpha beta and gamma affinities for muteins prior to run through model according to BLI data"""
+    # Adjust a affinities
+    input_params = mutdict.get(mut_name)
+
+    # change for unkVec instead of dict
+    rxntfr[:, 7] = input_params[0] * 0.6
+    rxntfr[:, 27] = rxntfr[:, 7] * 5.0
+
+    # Adjust b/g affinities
+    for jj in range(0, rxntfr.shape[0]):
+        bg_adjust = input_params[1] / rxntfr[jj, 10]
+        for ii in [8, 9, 10, 11, 12, 28, 29, 30, 31, 32]:
+            rxntfr[:, ii] = rxntfr[:, ii] * bg_adjust
```

Comment:
Don't think this is correct... you're looping over `jj` but assigning to the same place every time?